app/services/default_form_builder.rb

```diff
@@ -0,0 +1,32 @@
+class DefaultFormBuilder
+  FIELDS = [
```

Comment:
I agree, although I'm not convinced they belong in a config file. I'll move them to `Form`.